app/models/grda_warehouse/hud/project.rb

```diff
@@ -490,6 +490,11 @@ def rrh?
       self[self.class.project_type_column].to_i == 13
     end
 
+    scope :psh, -> { where(self.project_type_column => [3, 10]) }
+    def psh?
+      [3, 10].include? self[self.class.project_type_column]
```

Comment:
Two things here, can we make a method that returns the appropriate project type?  Essentially 
```
def project_type_to_use
  self[self.class.project_type_column]
end
```
and can we swap the order of the logic?  I find `project_type_to_use.in?([3, 10]) much easier to read.

Reply:
Probably also worth sticking [3, 10] into a method or constant, kinda like we do with `:ph`